src/views/TradingCompetition/index.tsx

```diff
@@ -71,14 +72,17 @@ const TradingCompetition = () => {
   const { profile, isLoading } = useProfile()
   const { isDark, theme } = useTheme()
   const tradingCompetitionContract = useTradingCompetitionContract()
```

Comment:
Do we need signer here? We can use simple rpc provider for this I think

Reply:
Yea, it's a good call. I will update it🙂